posthog/queries/base.py

```diff
@@ -64,7 +72,20 @@ def convert_to_comparison(trend_entity: List[Dict[str, Any]], filter, label: str
 """
 
 
-def handle_compare(filter, func: Callable, team: Team, **kwargs) -> List:
+class InsightResult(TypedDict):
```

Comment:
Nice!

Reply:
This seems unused though.

Reply:
Yeah not used in the file. I did update some of the interfaces here but it cased lots of mypy issues

I’ll move to trends.py(?)